src/tests/Mission.test.js

```diff
@@ -0,0 +1,19 @@
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import store from '../redux/configureStore';
+import Missions from '../components/Missions';
+
+describe('Missions component', () => {
+  describe('Use Jest snapshots to test the Missions component', () => {
+    it('renders Missions component as expected', async () => {
+      const missionsComponent = render(
+        <Provider store={store}>
```

Comment:
* Kindly fix the failing test 
![image](https://user-images.githubusercontent.com/52497246/146282252-322c435c-e4ff-4b69-9169-ad1859cd118c.png)

Reply:
@Davidosky007  it looks like you need to use npm install again, would you do it and try again, please?